<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обработка заявок на астрономические наблюдения</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}
QLabel {
  color: #464d55;
  font-weight: 600;
	border-radius: 20px;
}
QLabel#heading {
  color: #0f1925;
  font-size: 18px;
  margin-bottom: 10px;
}

QLabel#subheading {
  color: #0f1925;
  font-size: 12px;
  font-weight: normal;
  margin-bottom: 10px;
}
QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}

QLineEdit:focus {
  border: 1px solid #d0e3ff;
}

QLineEdit::placeholder {
  color: #767e89;
}
QPushButton {
  background-color:  rgb(200,200,200);
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  font-weight: 600;
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
  color: #fff;
}
QPushButton:pressed{
	 background-color:rgb(0, 150, 255);
}
QHeaderView::section {
    background-color: rgb(200,200,200);
}

QScrollBar:horizontal{
	border: none;
	background-color: rgb(255, 255, 255);
	border-radius: 0px;
}
QScrollBar::handle:horizontal{
	background-color:  rgb(200,200,200);
	border-radius: 10px;
}
QScrollBar::handle:horizontal:hover{
	background-color:  rgb(0, 85, 255);
}
QScrollBar::handle:horizontal:pressed{
	background-color:  rgb(0, 150, 255);

}
QScrollBar:vertical{
	border: none;
	background-color: rgb(255, 255, 255);
	border-radius: 0px;
}
QScrollBar::handle:vertical{
	background-color:  rgb(200,200,200);
	border-radius: 10px;
}

QScrollBar::handle:vertical:pressed{
	background-color:  rgb(0, 85, 255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1371</width>
      <height>561</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QHeaderView::section {
    background-color: rgb(200,200,200);
}

QScrollBar:horizontal{
	border: none;
	background-color: rgb(255, 255, 255);
	border-radius: 0px;
}
QScrollBar::handle:horizontal{
	background-color:  rgb(200,200,200);
	border-radius: 10px;
}
QScrollBar::handle:horizontal:hover{
	background-color:  rgb(0, 85, 255);
}
QScrollBar::handle:horizontal:pressed{
	background-color:  rgb(0, 150, 255);

}
QScrollBar:vertical{
	border: none;
	background-color: rgb(255, 255, 255);
	border-radius: 0px;
}
QScrollBar::handle:vertical{
	background-color:  rgb(200,200,200);
	border-radius: 10px;
}
/*QScrollBar::handle:vertical:hover{
	background-color:  rgb(0, 85, 255);
}
*/
QScrollBar::handle:vertical:pressed{
	background-color:  rgb(0, 85, 255)
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <column>
     <property name="text">
      <string>Время</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Эл. почта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип набл.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Р. старт</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>П. старт</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Прод-ть</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Смещение</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Величина</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип объекта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Комментарий</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="deny_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Отклонить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="approve_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Утвердить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>570</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-110</x>
      <y>-10</y>
      <width>1511</width>
      <height>831</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>6fadc29d97ab55aa5185d4116d5d10d3.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>576</y>
      <width>283</width>
      <height>69</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>2560px-Roscosmos_Corp_Brand_2015.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tableWidget</zorder>
   <zorder>deny_button</zorder>
   <zorder>approve_button</zorder>
   <zorder>refresh_button</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
